core/src/testFixtures/java/io/grpc/internal/AbstractTransportTest.java

```diff
@@ -212,6 +215,7 @@ public ServerStreamTracer newServerStreamTracer(String fullMethodName, Metadata
 
   @Before
   public void setUp() {
+    System.setProperty(GRPC_EXPERIMENTAL_SUPPORT_TRACING_MESSAGE_SIZES, "true");
```

Comment:
You should do this in the specific test, not here which is  the setup step for every test. We need to have 2 tests one that sets the property and one that doesn't. The test that does setProperty should have a try - finally block to System.clearProperty in the finally block as well.

Reply:
Since we would expect to have this flag for ~1 week and we want to get this in quickly (vs the larger revert). I suggest not testing at all with GRPC_EXPERIMENTAL_SUPPORT_TRACING_MESSAGE_SIZES=true (just use the default, whatever it is).